### PR TITLE
Fix/imports scaffolds

### PR DESCRIPTION
**Pull request by Karrenbelt**
## Proposed changes

Ensuring that the code can "run", that is to say that files can be imported, no `NameError`, etc.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: autonomy/cli/scaffold_fsm.py [hunk 1]
```diff
@@ -340,8 +355,13 @@ def _get_rounds_section(self) -> str:
             todo_abstract_round_cls = ""
             if abci_round_base_cls_name == ABSTRACT_ROUND:
                 todo_abstract_round_cls = "# TODO: replace AbstractRound with one of CollectDifferentUntilAllRound, CollectSameUntilAllRound, CollectSameUntilThresholdRound, CollectDifferentUntilThresholdRound, OnlyKeeperSendsRound, VotingRound"
+
+            base_name = abci_round_name.replace("Round", "")
+            round_id = _camel_case_to_snake_case(base_name)
             round_class_str = RoundFileGenerator.ROUND_CLS_TEMPLATE.format(
+                round_id=round_id,
```
Comment (Karrenbelt) [inline autonomy/cli/scaffold_fsm.py:362]: assigning default `round_id`

File: autonomy/cli/scaffold_fsm.py [hunk 2]
```diff
@@ -506,9 +532,14 @@ def _get_behaviours_section(self) -> str:
...
                 )
             )
+
+            behaviour_id = abci_behaviour_name.replace("Behaviour", "")
+            matching_round = abci_behaviour_name.replace("Behaviour", "Round")
             behaviour_class_str = BehaviourFileGenerator.BEHAVIOUR_CLS_TEMPLATE.format(
                 BehaviourCls=abci_behaviour_name,
                 BaseBehaviourCls=base_behaviour_cls_name,
+                behaviour_id=_camel_case_to_snake_case(behaviour_id),
+                matching_round=matching_round,
```
Comment (Karrenbelt) [inline autonomy/cli/scaffold_fsm.py:542]: assigning default `behaviour_id` and `matching_round`

File: autonomy/cli/scaffold_fsm.py [hunk 3]
```diff
@@ -610,10 +648,15 @@ def _get_base_payload_section(self) -> str:
...
 
+        tx_types = map(_camel_case_to_snake_case, self.base_names)
+        tx_types = [f'{tx_type.upper()} = "{tx_type}"' for tx_type in tx_types]
+        tx_types = indent("\n".join(tx_types), " " * 4).strip()
+
         return "\n".join(
             [
                 FILE_HEADER,
                 self.PAYLOADS_FILE.format(FSMName=self.abci_app_name),
+                self.TRANSACTION_TYPE_SECTION.format(tx_types=tx_types),
```
Comment (Karrenbelt) [inline autonomy/cli/scaffold_fsm.py:659]: creating default `TransactionType`

File: autonomy/cli/scaffold_fsm.py [hunk 4]
```diff
@@ -191,24 +192,29 @@ def author(self) -> str:
...
 
     @property
-    def rounds(self) -> Set[str]:
+    def all_rounds(self) -> Set[str]:
         """Rounds"""
         return self.dfa.states
 
     @property
-    def non_degenerate_rounds(self) -> Set[str]:
+    def rounds(self) -> Set[str]:
```
Comment (Karrenbelt) [inline autonomy/cli/scaffold_fsm.py:200]: addresses [comment](https://github.com/valory-xyz/open-autonomy/pull/1297#discussion_r964929869) in #1297

**Review by Adamantios — APPROVED** [review]
LGTM, let's create some constants for "Behaviour" and "Round" strings.

**Comment by Karrenbelt** [comment]
> 

added to #1289

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**